test/TestServerProjects/body-formdata-urlencoded/Generated/Models/Paths14Hl8BdFormsdataurlencodedPetAddPetidPostRequestbodyContentApplicationXWwwFormUrlencodedSchema.cs

```diff
@@ -0,0 +1,35 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+// <auto-generated/>
+
+#nullable disable
+
+namespace body_formdata_urlencoded.Models
+{
+    /// <summary> The Paths14Hl8BdFormsdataurlencodedPetAddPetidPostRequestbodyContentApplicationXWwwFormUrlencodedSchema. </summary>
+    internal partial class Paths14Hl8BdFormsdataurlencodedPetAddPetidPostRequestbodyContentApplicationXWwwFormUrlencodedSchema
```

Comment:
Hm, surprised this type is still generated even if internal.

Reply:
Can you follow up on this, please? I think M4 can avoid creating this schema.

Reply:
On it.

Reply:
https://github.com/Azure/autorest/issues/3897